<launch>
    
    <!-- Start laser_scan_multi_merger -->
    <node pkg="ira_laser_tools" name="laserscan_multi_merger"
    type="laserscan_multi_merger" output="screen">

    <param name="scan_destination_topic" value="/merged_scan"/>
    <param name="laserscan_topics" value="/hokuyoscan"/>
  </node>

</launch>
